.github/workflows/release.yaml

```diff
@@ -69,10 +69,10 @@ jobs:
         with:
           fetch-depth: 0
       - name: Select charmhub channel
-        uses: canonical/charming-actions/channel@1.0.3
```

Comment:
Question, this file seems to run `tox -e integration-ha`, but shouldn't it run all integration tests before releasing to charmhub, ie: `integration-test-lxd-mysql`, `integration-test-lxd-database`,  `integration-test-lxd-shared-db`, and `integration-test-lxd-db-router`

Reply:
yep! forgot to replicate after creating new tests. Good catch